tox.ini

```diff
@@ -1,8 +1,9 @@
 [tox]
 envlist=
-    py{38,39,310}-dj{32}-drf{312,313}-pyjwt{171,2}-tests
-    py{38,39,310}-dj{41,42}-drf313-pyjwt{171,2}-tests
-    py311-dj{41,42}-drf{313,314}-pyjwt{171,2}-tests
+    py{38,39,310}-dj32-drf{312,313,314,315}-pyjwt{171,2}-tests
+    py{38,39,310,311}-dj41-drf{314,315}-pyjwt{171,2}-tests
```

Comment:
I've removed DRF 3.12 and DRF 3.13 for Django >= 4.1 since [it started to be supported from 3.14](https://www.django-rest-framework.org/community/3.14-announcement/).

Also, I don't know if Django 4.1 should be removed since [it is no longer supported](https://docs.djangoproject.com/en/4.1/).

Reply:
hello, thanks for this patch. removing support for 4.1 isn't required as it is currently been supported by DRF 3.14.

Other than that, this good to go.

CC: @Andrew-Chen-Wang can you help make a review pass at this?